### PR TITLE
Add ODFE Kibana 1.10.1 upgrade instructions to ODFE docs for breaking changes on cookies security settings

### PR DESCRIPTION
**Pull request by peterzhuamazon**
*Issue #, if available:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/418

*Description of changes:*
Add ODFE Kibana 1.10.1 upgrade instructions to ODFE docs for breaking changes on cookies security settings

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/upgrade/1-10-1.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Upgrade to 1.10.1 (Kibana)
+parent: Upgrade
+nav_order: 3
+---
+
+# Upgrade to 1.10.1 (Kibana)
+
+Open Distro for Elasticsearch 1.10.1 includes a breaking change due to Security Kibana Plugin is re-implemented on top of Kibana's new plugin platform.
```
Comment (camerski) [inline docs/upgrade/1-10-1.md:10]: "Open Distro for Elasticsearch 1.10.1 includes a breaking change due to the Security Kibana plugin being re-implemented with Kibana's new plugin platform."

File: docs/upgrade/1-10-1.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+## Security Kibana Plugin
+
+If you:
+* Upgrade Kibana from any of the previous versions to 1.10.1
+* Use `Security Kibana Plugin` v1.10.1.1 and newer versions
```
Comment (camerski) [inline docs/upgrade/1-10-1.md:17]: "Use `Security Kibana Plugin` v1.10.1.1 or newer"

**Review by camerski — CHANGES_REQUESTED** [review]

File: docs/upgrade/1-10-1.md (hunk 1 above)
Comment (peterzhuamazon) [reply docs/upgrade/1-10-1.md]: Resolved

File: docs/upgrade/1-10-1.md (hunk 2 above)
Comment (peterzhuamazon) [reply docs/upgrade/1-10-1.md]: Resolved

**Comment by aetter** [comment]
Hey @peterzhuamazon, can you clarify the situation here? Is this only for people who installed the plugins standalone on top of the open source versions, people who used the all-in-one packages, or just anyone who's upgrading?

**Comment by peterzhuamazon** [comment]
> Hey @peterzhuamazon, can you clarify the situation here? Is this only for people who installed the plugins standalone on top of the open source versions, people who used the all-in-one packages, or just anyone who's upgrading?

It is for anyone that is upgrading.
We add this one as default in kibana.yml already.
However, we have noticed some users either copy their old config during upgrades.
Or tweaked their config for deb/rpm installation so package manager did not apply this new config.
Therefore, we think it is considered to be a breaking change at this point.
And have to add one of this page.

**Review by gaiksaya — APPROVED** [review]

**Review by bbarani — APPROVED** [review]

**Pull request merged**